utility/src/main/res/values/dimens.xml

```diff
@@ -6,7 +6,8 @@
   <!-- Space between the start of the line and the bullet in a bullet span. -->
   <dimen name="spacing_before_bullet">8dp</dimen>
   <!-- Space between bullet and the text in a bullet span. -->
-  <dimen name="spacing_before_text">20dp</dimen>
+  <dimen name="spacing_before_bullet_text">24dp</dimen>
+  <dimen name="spacing_before_numbered_text">28dp</dimen>
```

Comment:
Because of the calculation we are doing while drawing circle for bullets i.e it is taking (x,y) positions with bullet radius  and for drawing text we just appending string at x position and we are not giving any radius here so there is a bit gap difference. To cover this I used separate spacing to make it look approximately similar spacing.

Reply:
Bullets will be same for different font zooming and numbered items will be changed according to the font size. Please look at the screenshot below.
Emulator : Device Nexus 6 with largest font size.

<img width="173" alt="Screenshot_1649505206" src="https://user-images.githubusercontent.com/16301028/162573296-40b16cc0-29f3-4ff0-985d-99baebe14b8f.png">...<img width="173" alt="Screenshot_1649505190" src="https://user-images.githubusercontent.com/16301028/162573271-cdd8e742-d624-4f7a-b69b-e69b5803d152.png">

Reply:
Ah interesting. I think the spacing is fine as we can easily tweak this in the future.